packages/graphql/tests/integration/connections/alias.int.test.ts

```diff
@@ -0,0 +1,769 @@
+/*
```

Comment:
I can't believe with all of these tests you've written that I'm to ask for another...

But would be good to make sure you can alias a combination of nested fields - `edges`, `node`, `pageInfo` and `startCursor` for instance.